x-pack/plugins/infra/public/components/waffle/group_of_groups.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { InfraWaffleMapGroupOfGroups, InfraWaffleOptions } from '../../lib/lib';
+import { GroupName } from './group_name';
+import { GroupOfNodes } from './group_of_nodes';
+
+interface Props {
+  onDrilldown: () => void;
+  options: InfraWaffleOptions;
+  group: InfraWaffleMapGroupOfGroups;
+  formatter: (val: number) => string;
+}
+
+export const GroupOfGroups: React.SFC<Props> = props => {
+  return (
+    <Container>
+      <GroupName group={props.group} onDrilldown={props.onDrilldown} />
+      <Groups>
+        {props.group.groups.map(group => (
+          <GroupOfNodes
+            isChild={true}
+            key={group.id}
+            onDrilldown={props.onDrilldown}
+            options={props.options}
+            group={group}
+            formatter={props.formatter}
+          />
+        ))}
+      </Groups>
+    </Container>
+  );
+};
+
+export const Container = styled.div`
```

Comment:
Could we use a less generic name for this?

Reply:
What name would have made more sense here? 

Keep in mind I think that name is only visible in this file. It doesn't leak outside the file anywhere else, those styled components are isolated to the file. If you look at the page with the React dev tool it shows up under a different name. It was a mistake to `export` those containers, force of habit :D

Reply:
We could follow the naming convention you used in the other components, so it would be a `GroupOfGroupsContainer` in this case. (See also the review comment above.)